Main/ClientApp/src/Components/Drawer/MyCollectionsSection.tsx

```diff
@@ -148,8 +190,24 @@ export default function MyCollectionsSection() {
                       </ListItemIcon>
                       <ListItemText primary="Share" />
                     </ListItem>
-                    {/* <Divider /> */}
-                    {/* TODO: list of contributors */}
+                    {haveSharedCollections(collection.id) && <Divider />}
+                    {contributorsSharedCollections
+                      ?.filter((csc) => csc.collectionId === collection.id)
+                      .map((sc) => (
+                        <ListItem
+                          onClick={() =>
+                            onEditContributorClick(collection.id, sc.userId)
+                          }
+                          button
+                        >
+                          <ListItemAvatar>
+                            <Avatar>
+                              <Person />
```

Comment:
Consider displaying first letter of the user instead of a gray avatar for everyone.